main/feed_data.R

```diff
@@ -1,11 +1,17 @@
+library(plyr)
+library(readr)
+
 df <- data.frame(
   id = c(1,2,3,4,5),
   text = c("great","not great", "you jerk", "bananas", "orange jerks"),
   label = c(F, F, T, F, NA),
   training = c(T, T, T, T, F)
 )
+getwd()
+df <- read.csv("./src/data/both_t_data.csv")
```

Comment:
can you add a comment about what's in this file?

Reply:
ok

Reply:
I was thinking more semantics. Like, is this github issue data? pull requests? the same data that was in the NIER paper?